packages/Input/Input.jsx

```diff
@@ -1,8 +1,7 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import { childrenOfType } from 'airbnb-prop-types'
+import { componentWithName } from 'airbnb-prop-types'
 
-import Tooltip from '@tds/core-tooltip'
```

Comment:
If we have removed the only usage of `@tds/core-tooltip` in the Input package, then we should also remove it as a dependency from the package.json of that package. Same goes for Input, Select, TextArea, and FormField.